e2e/run.bash

```diff
@@ -9,4 +9,4 @@ echo ">>> Waiting for capsule pod to be ready for accepting requests"
 kubectl --namespace capsule-system wait --for=condition=ready --timeout=320s pod -l app.kubernetes.io/instance=capsule
 
 echo ">>> Starting test suite"
-bats -t "$(git rev-parse --show-toplevel)"/e2e/tests/06_*
```

Comment:
Oh, gosh: it's embarrassing, I forgot to remove this 🤣

Reply:
:)

> Could you split changes in 3 separate commits using the Semantic Commit Message style guide?

done :)

Reply:
:) i was wondering, if it this change was needed somehow, but checked previous version and understood that it was just a typo